Berksfile.lock

```diff
@@ -1,33 +0,0 @@
-DEPENDENCIES
```

Comment:
Why was this removed?

Reply:
Partly because it's default with Chef (per other comment: when running `chef generate cookbook`), but also because of (a.) wrapper cookbooks, and/or (b.) policyfiles ... bit of a shift in the community. I can back this out, if preferred.

Reply:
I don't know enough about chef to comment but we need a way to have reproducible builds on every machine (CI/dev) so this can't be removed until that problem has been solved. It would be outside the scope of this change either way.

Reply:
Understandable on the reproducibility angle ... any chance of getting this cookbook hooked up to something like travis-ci for builds (ex: have a look at https://github.com/chef-cookbooks/nginx)
All that being said, because metadata.rb has pessimistic versioning, there's no guarantee that subsequent `berks update` runs will be identical anyway.
I'll update the PR for now.

Reply:
We have gone down the avenue of using travis for Sonatype projects and it usually runs into headaches maintaining an internal Jenkins build and external travis build. I need to think more about the benefit of feedback to external contributors against the maintenance requirements.